Source/NSBrowser.m

```diff
@@ -407,15 +415,15 @@ - (NSInteger) selectedRowInColumn: (NSInteger)column
   return [matrix selectedRow];
 }
 
-/**<p>Selects the cell at index <var>row</var> in the column identified by 
-   index <var>column</var>. If the delegate method 
+/**<p>Selects the cell at index <var>row</var> in the column identified by
+   index <var>column</var>. If the delegate method
    -browser:selectRow:inColumn: is implemented, this is its responsability
    to select the cell. This method adds a NSBrowser column if needed
-   and deselects other selections if the browser does not allows multiple 
-   selection.</p><p>See Also: -loadedCellAtRow:column: 
+   and deselects other selections if the browser does not allows multiple
+   selection.</p><p>See Also: -loadedCellAtRow:column:
    -browser:selectRow:inColumn: [NSMatrix-selectCellAtRow:column:]</p>
  */
-- (void) selectRow: (NSInteger)row inColumn: (NSInteger)column 
+- (void) selectRow: (NSInteger)row inColumn: (NSInteger)column
```

Comment:
@fredkiefer I believe this is the method where a multiple selection causes the last column to be unloaded.  The method you commented on earlier is responsible for getting the selection of the previous column so that the current column knows what to display (based on the children it retrieves).